14/Dockerfile.fedora

```
@@ -58,6 +58,7 @@ RUN yum -y module enable nodejs:$NODEJS_VERSION && \
 
 # Copy the S2I scripts from the specific language image to $STI_SCRIPTS_PATH
 COPY ./s2i/bin/ $STI_SCRIPTS_PATH
+RUN chmod +x $STI_SCRIPTS_PATH/init-wrapper
```

It looks like environment variable is not defined.
```suggestion
RUN chmod +x /usr/libexec/s2i/init-wrapper
```

Error from build.log https://artifacts.dev.testing-farm.io/153479fc-1c57-4316-9abe-5979c2081982/
```bash
STEP 7/13: COPY ./s2i/bin/ /usr/libexec/s2i
--> 636dec162dc8
STEP 8/13: RUN chmod +x $STI_SCRIPTS_PATH/init-wrapper
chmod: cannot access '/init-wrapper': No such file or directory
Error: building at STEP "RUN chmod +x $STI_SCRIPTS_PATH/init-wrapper": while running runtime: exit status 1
errexit on line 33, common/build.sh
errexit on line 33, common/build.sh
```

---

Should I update across the whole repo this command `RUN chmod +x $STI_SCRIPTS_PATH/init-wrapper` with your suggestion `RUN chmod +x /usr/libexec/s2i/init-wrapper`, to be consistent?

---

Also I have the impression that this fails quite random. as there are no significant differences between the images that fail the one that dont. 
Also, I'm wondering why this fails on the `RUN chmod +x $STI_SCRIPTS_PATH/init-wrapper` and not on the `COPY ./s2i/bin/ $STI_SCRIPTS_PATH` which is one command earlier.

---

I see that this one is not the one that fails.